about.html

```diff
@@ -0,0 +1,142 @@
+            </div>
+            <div id="contact-container">
+                <p id="" class="text-center contact-text">Please contact us per Email for any further questions about CC Global Summit 2023!</p>
+                <a href="" class="text-center fw-bold d-block text-decoration-none text-black"><span id="contact-email">techconference2023@newyork.org</span></a>
+                <div class="highlight"></div>
+            </div>
+        </section>
+    </main>
+    <!-- LOGO SECTION   -->
+    <section id="logo-section" class="container-fluid mt-5">
+            <h2 class="logo-text">Tech Innovation Conference 2023 Logo</h2>
+            <div class="line"></div>
+            <p id="" class="text-center mt-3 contact-text d-block d-md-none">The logo of Tech Conference 2023 was decided through the logo competition from 8.June to 7.July.</p>
+            <p id="" class="text-center mt-3 contact-text d-none d-md-block">The logo of Tech Conference 2023 was decided through the logo competition from 8.June to 7.July.
+                After online public voting and Selection Committee's judging process, the logo from Naresh Agrawal won the prize. After consulation with Mr.Agrawal we optimized the prize-winning-logo for
+                this year's summit as following.
+            </p>
+
+            <div class="tech-logo-container">
+                <img src="./images/tech-conference-logo.webp" alt="" id="tech-logo">
+            </div>
+    </section>
+    <!-- PAST SUMMIT SECTION  -->
+    <section id="past-summit" class="container-fluid mt-5">
+        <h2 class="logo-text">See the Past Tech Conference</h2>
+        <div class="line"></div>
+        <p id="" class="text-center mt-3 contact-text">Take a look at the last two Tech Conference which took place in Buneo Aires and in Warsaw.</p>
+        <div class="row gy-2 mt-md-2">
+            <div class="col-9 mx-auto col-md-5 past-summit-card" id="summit-card-one">
+                <h2 class="year text-center text-white">2018</h2>
+                <p class="text-center summit-card-text text-white">Tech Conference 2018 in Hongkong</p>
+            </div>
+            <div class="col-9 mx-auto col-md-5 past-summit-card" id="summit-card-two">
+                <h2 class="year text-center text-white">2016</h2>
+                <p class="text-center summit-card-text text-white">Tech Conference 2016 in Thailand</p>
+            </div>
+        </div>
+    </section>
+    <!-- FOOTER SECTION  -->
+    <footer id="footer-section" class="container-fluid mt-5 ">
+        <div class="footer-container d-flex align-items-center">
+            <!-- <img src="./images/tech-conference-logo.webp" class="me-2 flex-grow-1" alt=""> -->
+            <i class="fa-solid fa-copyright copyright-icon-about"></i>
+            <h5  class="text-center">Tech Conference <br>Hongkong</h5>
+            <p class=" text-center ">2023 Tech Conference Hongkong. Some Rights Resered.</p>
+        </div>
+    </foot>
```

Comment:
- [ ] Kindly make sure to add the sponsor's section above the footer as it should be displayed in the mobile version and not in the desktop version.
- Your design:
  ![image](https://github.com/HtetWaiYan7191/capstone-project/assets/104968236/9b4523e5-1646-4026-8dc9-54b26fc41099)
- Original design:
 ![image](https://github.com/HtetWaiYan7191/capstone-project/assets/104968236/46699d66-bdaa-4a41-b9e6-e30df9103e7c)